web/app/components/base/chat/embedded-chatbot/header.tsx

```diff
@@ -1,10 +1,15 @@
-import type { FC } from 'react'
+import type { FC, SVGProps } from 'react'
 import React from 'react'
 import { useTranslation } from 'react-i18next'
 // import AppIcon from '@/app/components/base/app-icon'
-import { ReplayIcon } from '@/app/components/app/chat/icon-component'
 import Tooltip from '@/app/components/base/tooltip'
 
+export const ReplayIcon = ({ className }: SVGProps<SVGElement>) => (
```

Comment:
`import { ReplayIcon } from '@/app/components/app/chat/icon-component'`

Should we use this icon here?

Reply:
replaced with new icon yet